app/forms/hub/take_action_form.rb

```diff
@@ -9,9 +9,9 @@ class TakeActionForm < Form
                   :internal_note_body,
                   :action_list,
                   :current_user,
-                  :client,
-                  :current_state
-    validates_presence_of :current_state
+                  :client
+    attr_accessor :state_transition, :status
```

Comment:
is this `state_transition` needed? seems like `:current_state` -> `:status` was the necessary part